sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharpExplodedGraph.cs

```diff
@@ -393,6 +393,8 @@ protected override void VisitInstruction(ExplodedGraphNode node)
 
                 case SyntaxKind.CheckedExpression:
                 case SyntaxKind.UncheckedExpression:
+
+                case SyntaxKind.BreakStatement:

```

Comment:
this is probably a wrong fix, the problem is the `break` being part of the instructions of the `JumpBlock`